src/main/scala/com/ibm/couchdb/api/builders/GetManyDocumentsQueryBuilder.scala

```diff
@@ -160,13 +213,69 @@ case class GetManyDocumentsQueryBuilder(
   }
 
   private def queryWithoutIds[Q: R](ps: Map[String, String]): Task[Q] = {
-    postQuery[Q](client, db, s"/$db/_all_docs", ps)
+    QueryUtils.query[Q](client, db, s"/$db/_all_docs", ps)
   }
 
   private def queryByIds[Q: R](ids: Seq[String], ps: Map[String, String]): Task[Q] = {
     if (ids.isEmpty)
       Res.Error("not_found", "No IDs specified").toTask
     else
-      queryByIds[String, Q](client, db, s"/$db/_all_docs", ids, ps)
+      QueryUtils.queryByIds[String, Q](client, db, s"/$db/_all_docs", ids, ps)
+  }
+}
+
+object GetManyDocumentsQueryBuilder {
+
+  trait BuildReady[T] {
```

Comment:
Adding implementations of this trait for the query strategies could simplify the implicits (especially if QueryBasic and QueryIncludeDocs get merged).